packages-exp/auth-types-exp/index.d.ts

```diff
@@ -200,7 +203,6 @@ export abstract class OAuthCredential implements AuthCredential {
   constructor();
 
   toJSON(): object;
-  fromJSON(json: object | string): OAuthCredential | null;
```

Comment:
what about these static methods?

Reply:
They just need to be defined on every `AuthCredential` implementation. Unfortunately there's no way in TS to define a static method that's always available. This kinda makes sense though-- you always need a reference to the actual class (not just an interface) in order to call a static method.